src/pages/docs/api/test-environments.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Test Environments
+description: Manage test environments
+---
+
+Manage test environments for your products.
+
+> **Prerequisites**: You need a product before creating test environments. See [Products](/docs/api/products) to get or create a product. For mobile app tests, you may need to upload a binary app first (see [Binary Apps](/docs/api/binary-apps)).
+
+## List test environments
+
+Retrieve all test environments for a specific product.
+
+**Endpoint:** `GET /products/{product_id}/test_environments`
+
+**Parameters:**
+
+- `product_id` (number, required) - ID of the Product
+
+**Example Request:**
+
+{% code language="bash" showLineNumbers=true %}
+```bash
+curl -X GET "https://api.test.io/customer/v2/products/1/test_environments" \
+  -H "Authorization: Token YOUR_API_TOKEN"
+```
+{% /code %}
+
+**Response:** `200 OK`
+
+Returns an array of test environment objects.
+
+## Get test environment
+
+Retrieve a specific test environment by ID.
+
+**Endpoint:** `GET /products/{product_id}/test_environments/{test_environment_id}`
+
+**Parameters:**
+
+- `product_id` (number, required) - ID of the Product
+- `test_environment_id` (number, required) - ID of the Test Environment
+
+**Example Request:**
+
+{% code language="bash" showLineNumbers=true %}
+```bash
+curl -X GET "https://api.test.io/customer/v2/products/1/test_environments/42" \
+  -H "Authorization: Token YOUR_API_TOKEN"
+```
+{% /code %}
+
+**Response:** `200 OK`
+
+Returns the test environment object.
+
+## Create test environment
```

Comment:
We need to also mention the large file upload in this section for people who want to upload large file